components/about/contactForm.js

```diff
@@ -1,22 +1,131 @@
-import {
-  Box,
-  Text,
-  Link
-} from '@chakra-ui/react'
+/* global fetch */
+import { useState } from 'react'
+import { Input, Textarea, Button, FormControl, FormLabel, FormHelperText, useToast } from '@chakra-ui/react'
 
 const ContactForm = () => {
+  const [name, setName] = useState('')
+  const [email, setEmail] = useState('')
+  const [subject, setSubject] = useState('')
+  const [message, setMessage] = useState('')
+
+  const toast = useToast()
+  const showToast = ({ title, description, status, duration }) => (
+    toast({
+      title,
+      description,
+      status,
+      duration,
+      isClosable: true
+    })
+  )
+
+  const submitHandler = (e) => {
+    e.preventDefault()
+
+    const data = {
+      name,
+      email,
+      subject,
+      message
+    }
+
+    fetch('/api/contact', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json, text/plain, */*',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(data)
+    }).then((res) => {
+      if (res.status === 200) {
+        setName('')
+        setEmail('')
+        setSubject('')
+        setMessage('')
+        showToast({
+          title: 'Message Sent',
+          description: "We'll be in touch!",
+          status: 'success',
+          duration: '8000'
+        })
+      } else {
+        // TODO: Add better error handling
+        console.log('Response:', res)
+        showToast({
+          title: 'Error sending message',
+          description: `HTTP Response Code: ${res.status}`,
+          status: 'error',
+          duration: '12000'
+        })
+      }
+    }).catch((err) => {
+      // TODO: Add better error handling
```

Comment:
can you remove these console logs? also - you set the description of the toast to err, should it be err.message?